main/serve.js

```diff
@@ -37,23 +55,20 @@ let server = pkg.Server.create({
 });
 
 const logger = foam.log.ConsoleLogger.create();
-let credentials = null;
-try {
-  credentials = require('../.local/credentials.json');
-} catch (e) {
-  logger.warn(`No Cloud Datastore credentails found. Using read-only copy of
-                   local data snapshot`);
-}
-const daoContainer = credentials ? pkg.DatastoreContainer.create({
-  mode: pkg.DatastoreContainerMode.WEB_SERVICE,
-  gcloudAuthEmail: credentials.client_email,
-  gcloudAuthPrivateKey: credentials.private_key,
-  gcloudProjectId: credentials.project_id,
-  logger: logger,
-}) : pkg.DAOContainer.create({
-  logger: logger,
+const modeString = process.argv[2];
+const mode = pkg.JsonDAOContainerMode.VALUES.filter(function(value) {
+  return value.name === modeString;
+})[0] || pkg.JsonDAOContainerMode.LOCAL;
```

Comment:
I think erroring out on bad input is better than defaulting to local.

Reply:
Done.